src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1460,6 +1464,40 @@ public function configureProductPaginationAction(Request $request): JsonResponse
         return $this->json(null, Response::HTTP_NO_CONTENT);
     }
 
+    /**
+     * Method for downloading customization picture
+     *
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param int $type
+     * @param string $name
+     * @param string $value
+     *
+     * @return BinaryFileResponse
+     */
+    public function displayCustomizationImageAction(int $type, string $name, string $value)
+    {
+        $uploadDir = $this->get('prestashop.adapter.legacy.context')->getUploadDirectory();
```

Comment:
I'm not sure how to deal with exceptions in this action, since it's a download feature (there's no place to display any error message)